retrievalmarket/rtvllog/retrieval_log.go

```diff
@@ -58,6 +59,30 @@ func (r *RetrievalLog) OnQueryEvent(evt retrievalmarket.ProviderQueryEvent) {
 		"status", evt.Response.Status,
 		"msg", evt.Response.Message,
 		"err", evt.Error)
+
+	// Log failures to DB
+	st := &RetrievalDealState{
+		UnsealPrice: evt.Response.UnsealPrice,
+		Message:     evt.Response.Message,
+	}
+	if evt.Error != nil {
+		st.Status = fmt.Sprintf("failed to respond to the query: %s", evt.Error.Error())
+	} else {
+		if evt.Response.Status == retrievalmarket.QueryResponseUnavailable {
+			st.Status = "retrieval query offer was unavailable"
+		}
+		if evt.Response.Status == retrievalmarket.QueryResponseError {
+			st.Status = "retrieval query offer errored"
+		}
+	}
```

Comment:
The status should just be a short string like "Failed" or "Unavailable". The Message contains the actual error message.

Reply:
I will update the strings